lib/ex_unit/lib/ex_unit/callbacks.ex

```diff
@@ -234,28 +234,33 @@ defmodule ExUnit.Callbacks do
   end
 
   @doc false
-  def __merge__(_mod, context, :ok) do
+  def __merge__(mod, context, value) do
+    __merge__(mod, context, value, value)
+  end
+
+  @doc false
+  defp __merge__(_mod, context, :ok, _original_value) do
```

Comment:
Noted, thanks for the explanation! I've renamed `__merge__/4` to `merge/4` in adf2ee61dd8b61f1966cdeca1e791994e905c886. :)